contrib/node/examples/src/node/hello-test/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Actually, if there is no node_modules installed here. It's impossible to generate a yarn.lock file and it is impossible to use yarn as the package manager.